tests/manage/rgw/test_host_node_failure.py

```diff
@@ -72,6 +73,17 @@ def test_rgw_host_node_failure(
         if noobaa_pod_node is None:
             assert False, "Could not find the NooBaa DB pod"
 
+        # Validate if RGW pod and noobaa-db are hosted on same node else skip test
+        rgw_pod_node_list = []
+        for rgw_pod in rgw_pod_obj:
+            rgw_pod_node_list.append(rgw_pod.get().get("spec").get("nodeName"))
+        if set(rgw_pod_node_list).intersection(noobaa_pod_node.name):
+            pytest.skip(
+                "RGW pod and NooBaa DB are not hosted on same node. "
+                f"RGW pod hosted on nodes: {rgw_pod_node_list} "
+                f"NooBaa DB pod hosted on node: {noobaa_pod_node.name}"
+            )
```

Comment:
We are skipping the test if we dont find both rgw and noobaa db pod running on the same node. 
There can be high chances that this test gets skipped frequently.